src/synthesizer/python/mod.rs

```diff
@@ -0,0 +1,645 @@
+use crate::code::{CodeBuffer, IndentOptions};
+use crate::ir::conditions::ConditionIr;
+use crate::ir::importer::ImportInstruction;
+use crate::ir::mappings::MappingInstruction;
+use crate::ir::outputs::OutputInstruction;
+use crate::ir::reference::{Origin, PseudoParameter, Reference};
+use crate::ir::resources::{ResourceInstruction, ResourceIr};
+use crate::ir::CloudformationProgramIr;
+use crate::parser::lookup_table::MappingInnerValue;
+use indexmap::IndexMap;
+use std::borrow::Cow;
+use std::io;
+use std::rc::Rc;
+use voca_rs::case::camel_case;
+
+use super::Synthesizer;
+
+const INDENT: Cow<'static, str> = Cow::Borrowed("  ");
+
+pub struct Python {
+    // TODO: Put options in here for different outputs in Python
```

Comment:
Can you clarify what you mean by this TODO?

Reply:
Unfortunately I can't. I copied as much of the Typescript synthesizer structure as I possibly could, and that meant needing this Python struct with nothing in it that we just impl. Romain had that comment so I added it too lol

Reply:
went ahead and just deleted it.